psql2mysql/__init__.py

```diff
@@ -161,7 +161,15 @@ def writeTableRows(self, table, rows):
         )
 
     def clearTable(self, table):
-        self.connection.execute(table.delete())
+        LOG.debug("Purging table: %s", table.name)
+        while True:
+            # DELETE ... LIMIT is MySQL specific and there is no way to
+            # express it in SQLalchemy other than using a raw query.
+            res = self.connection.execute(
+                'DELETE FROM `%s` LIMIT 10000' % table.name
```

Comment:
Why 10000? Isn't there a chance that user has it set to lower number by some config option?

Reply:
Actually the plan was to mark this WIP and adapt it to use the same chunksize as for the chunked writing passed in via parameter in (see #25 ).

Reply:
Makes sense